addons/spreadsheet/static/src/pivot/plugins/pivot_core_plugin.js

```diff
@@ -273,76 +274,17 @@ export default class PivotCorePlugin extends CorePlugin {
      */
     _insertPivot(sheetId, anchor, id, table) {
```

Comment:
could we take this occasion to convert anchor to a `Position: {col, row}` Object ?